app/models/discussion.rb

```diff
@@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
-  after_save :update_subscriptions
```

Comment:
It's one to save the comment, and another to touch the discussion. It's unnecessary, but it's also not problematic. And what would be the alternative?